indico/modules/events/abstracts/operations.py

```diff
@@ -62,3 +63,41 @@ def delete_abstract(abstract, delete_contrib=False):
     logger.info('Abstract %s deleted by %s', abstract, session.user)
     abstract.event_new.log(EventLogRealm.management, EventLogKind.negative, 'Abstracts',
                            'Abstract "{}" has been deleted'.format(abstract.title), session.user)
+
+
+def judge_abstract(abstract, abstract_data, judgment, merge_persons=False):
+    if judgment not in AbstractAction:
+        raise ValueError("{} is not a valid judgment".format(judgment))
+    abstract.populate_from_dict(abstract_data)
```

Comment:
Not sure if this is a good idea. I'd rather populate the appropriate fields in the various if bodies. Otherwise you need to look at the form what's set for a specific action instead of immediately seeing it here...